pyqt5_ui.py

```diff
@@ -0,0 +1,102 @@
+import sys
+from PyQt5.QtWidgets import QApplication, QMainWindow, QFrame, QMessageBox
+from game import Game
+from PyQt5.QtGui import QPainter, QColor
+
+
+class CrushUI(QMainWindow):
+    def __init__(self):
+        super().__init__()
+        red = (255, 0, 0)
```

Comment:
declare colours as global constants. Can also be done in a separate file colours.py

Reply:
I put the colors in a separate file